Transfer/Sources/TransferAmountCell.swift

```diff
@@ -109,41 +109,71 @@ final class TransferAmountCell: UITableViewCell {
 
 extension TransferAmountCell: UITextFieldDelegate {
     func textFieldDidEndEditing(_ textField: UITextField) {
-        enteredAmountHandler?(amountTextField.text ?? "")
+        guard var currentText = textField.text else {
+            enteredAmountHandler?("")
+            return
+        }
+        let decimalPointSymbol = "."
```

Comment:
why do this?

Reply:
to remove redundant point at the end of a number at the end of editing

Reply:
I get it but my question was why do we need this? If there's . in the end and the destination currency allows it that's fine. If destination currency does not allow . then api will give an error